openapi/components/schemas/Storefront/PreviewPurchase.yaml

```
@@ -52,14 +52,20 @@ properties:
     items:
       type: object
       properties:
-        planId:
-          description: The item plan identifier string.
-          allOf:
-            - $ref: ../ResourceId.yaml
-        productId:
-          description: The item product identifier string.
-          allOf:
-            - $ref: ../ResourceId.yaml
+        type:
+          description: Type of the item.
+          type: string
+          enum:
+            - debit
+            - credit
+        description:
+          description: Description of the item.
+          type: string
+        unitPriceAmount:
```

That's confusing naming, why not make to similar to InvoiceItem?

```suggestion
        unitPrice:
```

---

Actually this is a typo, thank you!